src/main/java/ch/njol/skript/expressions/ExprExperience.java

```diff
@@ -72,6 +73,8 @@ protected Experience[] get(final Event e) {
 			return new Experience[] {new Experience(((ExperienceSpawnEvent) e).getSpawnedXP())};
 		else if (e instanceof BlockBreakEvent)
 			return new Experience[] {new Experience(((BlockBreakEvent) e).getExpToDrop())};
+		else if (e instanceof PlayerExpChangeEvent)
+			return new Experience[] {new Experience(((PlayerExpChangeEvent) e).getAmount())};
 		else
 			return new Experience[0];
```

Comment:
Chained if statements should have curly braces

Reply:
I think this whole class needed a refreshment therefore it can stay as is for now and in another PR the whole classes gets updated

Or if the author doesn't mind they can do it in this PR

Reply:
I think this whole class needed a refreshment therefore it can stay as is for now and in another PR the whole classes gets updated

Or if the author doesn't mind they can do it in this PR